indico/modules/events/editing/operations.py

```diff
@@ -231,6 +232,14 @@ def delete_tag(tag):
     db.session.delete(tag)
 
 
+def create_new_file_type(event, name, extensions=[], allow_multiple_files=False, required=False, publishable=False):
+    file_type = EditingFileType(name=name, extensions=extensions, allow_multiple_files=allow_multiple_files,
```

Comment:
What about `= EditingFileType()` and then using `populate_from_dict` like below when editing?